docs/docs/reference/docgen/javascript/auth/content/hasura-auth-client/content/11-on-auth-state-changed.mdx

```diff
@@ -0,0 +1,28 @@
+---
+# ⚠️ AUTO-GENERATED CONTENT. DO NOT EDIT THIS FILE DIRECTLY! ⚠️
+title: onAuthStateChanged()
+sidebar_label: onAuthStateChanged()
+slug: /reference/javascript/auth/on-auth-state-changed
+description: Use `nhost.auth.onAuthStateChanged` to add a custom function that runs every time the authentication status of the user changes. E.g. add a custom function that runs every time the authentication status changes from signed-in to signed-out.
```

Comment:
Why is this file not already generated and existing in our `main` branch?

I would suspect this file to not be part of the diff of this PR since I didn't modify anything related to `hasura-auth-client`.